adapter/src/test/java/org/jvalue/ods/adapterservice/datasource/model/DatasourceTest.java

```diff
@@ -61,6 +64,35 @@ public void testSerialization() throws ParseException {
         result.get("protocol").get("parameters").get("location").textValue());
   }
 
+  @Test
+  public void testFillQueryParameters() throws ParseException {
```

Comment:
IMO you should remove this test and make the fillQueryParameters() method private. I think it is covered by the test below.
However, if you prefer keeping it, that would be fine with me as well.

Reply:
I would like to keep it. Perhaps more logic will be added to both methods in the future. Then it would be good to be able to test them separately.

Reply:
this convinces me